include/zephyr/posix/sys/stat.h

```diff
@@ -114,7 +114,7 @@ struct stat {
 #define st_mtime st_mtim.tv_sec
 #define st_ctime st_ctim.tv_sec
 #if defined(__linux) && defined(__x86_64__)
-	__uint64_t __glibc_reserved[3];
+	uint64_t __glibc_reserved[3];
```

Comment:
Yeah, that was a new one - it's because we now support running on `native_sim` (and this is pulled in via picolibc / pthreads headers).

The Great POSIX Header Cleanup (TM) will need to happen soon, I think.